### PR TITLE
Batch update of alt text and title attributes for images in md/mdx files across repo

### PR DESCRIPTION
**Pull request by jonathanalgar**
**Description:** Batch update of alt text and title attributes for images in `md` & `mdx` files across the repo using [alttexter](https://github.com/jonathanalgar/alttexter)/[alttexter-ghclient](https://github.com/jonathanalgar/alttexter-ghclient) (built using LangChain/LangSmith).

**Limitation:** cannot update `ipynb` files because of [this issue](https://github.com/langchain-ai/langchain/pull/15357#issuecomment-1885037250). Can revisit when Docusaurus is bumped to v3.

I checked all the generated alt texts and titles and didn't find any technical inaccuracies. That's not to say they're _perfect_, but a lot better than what's there currently.

[Deployed](https://langchain-819yf1tbk-langchain.vercel.app/docs/modules/model_io/) image example:

chrome_yZQ7BF2GTj

You can see LangSmith traces for all the calls out to the LLM in the PRs merged into this one:

* https://github.com/jonathanalgar/langchain/pull/6
* https://github.com/jonathanalgar/langchain/pull/4
* https://github.com/jonathanalgar/langchain/pull/3

I didn't add the following files to the PR as the images already have OK alt texts:

* https://github.com/langchain-ai/langchain/blob/27dca2d92fbd9c956a1eac7cf885a0ed060ca9ff/docs/docs/integrations/providers/argilla.mdx#L3
* https://github.com/langchain-ai/langchain/blob/27dca2d92fbd9c956a1eac7cf885a0ed060ca9ff/docs/docs/integrations/providers/apify.mdx#L11

### CODE REVIEW

**Comment by jonathanalgar** [comment]
@efriis I'm not sure why the changes in the ipynb alt texts are breaking the build 🤔 for example:

```
SyntaxError: /home/ja/langchain-2/_dist/docs/modules/chains/document/map_reduce.md: Unterminated string constant. (16:13)
  14 | them to an LLM). This compression step is performed recursively if
  15 | necessary.`}</p>
> 16 | <p><img alt={"Diagram illustrating the map reduce documents chain with Map and
     |              ^
  17 | Reduce steps, including prompts and model
  18 | processing."} src={require("!/home/ja/langchain-2/_dist/node_modules/url-loader/dist/cjs.js?limit=10000&name=assets/images/[name]-[contenthash].[ext]&fallback=/home/ja/langchain-2/_dist/node_modules/file-loader/dist/cjs.js!./../../../../static/img/map_reduce.jpg").default} width="1916" height="616" /></p>
  19 | <h2 {...{"id":"recreating-with-lcel"}}>{`Recreating with LCEL`}</h2>
SyntaxError: /home/ja/langchain-2/_dist/docs/modules/chains/document/refine.md: Unterminated string constant. (21:13)
  19 | when documents frequently cross-reference one another or when a task
  20 | requires detailed information from many documents.`}</p>
> 21 | <p><img alt={"Diagram illustrating the Refine documents chain process with loops and
     |              ^
  22 | model interaction."} src={require("!/home/ja/langchain-2/_dist/node_modules/url-loader/dist/cjs.js?limit=10000&name=assets/images/[name]-[contenthash].[ext]&fallback=/home/ja/langchain-2/_dist/node_modules/file-loader/dist/cjs.js!./../../../../static/img/refine.jpg").default} width="3616" height="1568" /></p>
  23 | <h2 {...{"id":"recreating-with-lcel"}}>{`Recreating with LCEL`}</h2>
  24 | <p>{`With `}<a parentName="p" {...{"href":"/docs/expression_language"}}>{`LangChain Expression Language`}</a>{` we can
```

the md syntax is identical as before

**Comment by jonathanalgar** [comment]
> @efriis I'm not sure why the changes in the ipynb alt texts are breaking the build 🤔 for example:
> 
> ```
> SyntaxError: /home/ja/langchain-2/_dist/docs/modules/chains/document/map_reduce.md: Unterminated string constant. (16:13)
>   14 | them to an LLM). This compression step is performed recursively if
>   15 | necessary.`}</p>
> > 16 | <p>      |              ^
>   17 | Reduce steps, including prompts and model
>   18 | processing."} src={require("!/home/ja/langchain-2/_dist/node_modules/url-loader/dist/cjs.js?limit=10000&name=assets/images/[name]-[contenthash].[ext]&fallback=/home/ja/langchain-2/_dist/node_modules/file-loader/dist/cjs.js!./../../../../static/img/map_reduce.jpg").default} width="1916" height="616" /></p>
>   19 | <h2 {...{"id":"recreating-with-lcel"}}>{`Recreating with LCEL`}</h2>
> SyntaxError: /home/ja/langchain-2/_dist/docs/modules/chains/document/refine.md: Unterminated string constant. (21:13)
>   19 | when documents frequently cross-reference one another or when a task
>   20 | requires detailed information from many documents.`}</p>
> > 21 | <p>      |              ^
>   22 | model interaction."} src={require("!/home/ja/langchain-2/_dist/node_modules/url-loader/dist/cjs.js?limit=10000&name=assets/images/[name]-[contenthash].[ext]&fallback=/home/ja/langchain-2/_dist/node_modules/file-loader/dist/cjs.js!./../../../../static/img/refine.jpg").default} width="3616" height="1568" /></p>
>   23 | <h2 {...{"id":"recreating-with-lcel"}}>{`Recreating with LCEL`}</h2>
>   24 | <p>{`With `}<a parentName="p" {...{"href":"/docs/expression_language"}}>{`LangChain Expression Language`}</a>{` we can
> ```
> 
> the md syntax is identical as before

identified issue, working on a fix later this evening!

**Comment by jonathanalgar** [comment]
Removed `ipynb` files from PR—there is an issue with the alt text getting wrapped in the markdown cells (default filters of the stable version of Quarto). This is handled by Docusaurus v3+ but not the Docusaurus that the build uses (2.4.3). Full discussion: https://github.com/quarto-dev/quarto-cli/discussions/8131

tl;dr: I'll open a new PR with the alt texts for the `ipynb` files once Docusaurus is bumped to v3+.
